<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutMySelf</class>
 <widget class="QDialog" name="AboutMySelf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>关于</string>
  </property>
  <widget class="QLabel" name="MyNameLable">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>xiaomaotou31</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="MyheadLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="UrlLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="GithubLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>54</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
